include/z64save.h

```diff
@@ -690,11 +690,11 @@ typedef enum {
 #define WEEKEVENTREG_16_02 PACK_WEEKEVENTREG_FLAG(16, 0x02)
 #define WEEKEVENTREG_16_04 PACK_WEEKEVENTREG_FLAG(16, 0x04)
 #define WEEKEVENTREG_16_08 PACK_WEEKEVENTREG_FLAG(16, 0x08)
-#define WEEKEVENTREG_16_10 PACK_WEEKEVENTREG_FLAG(16, 0x10)
+#define WEEKEVENTREG_TALKED_KOUME_INJURED PACK_WEEKEVENTREG_FLAG(16, 0x10)
 #define WEEKEVENTREG_16_20 PACK_WEEKEVENTREG_FLAG(16, 0x20)
 #define WEEKEVENTREG_16_40 PACK_WEEKEVENTREG_FLAG(16, 0x40)
 #define WEEKEVENTREG_16_80 PACK_WEEKEVENTREG_FLAG(16, 0x80)
-#define WEEKEVENTREG_17_01 PACK_WEEKEVENTREG_FLAG(17, 0x01)
+#define WEEKEVENTREG_TALKED_EMPTY_BOAT_CRUISE PACK_WEEKEVENTREG_FLAG(17, 0x01)
```

Comment:
Could you explain what this means? Is this talking to the empty stall in the boat house before Koume returns? Would this mean this is set by the msgeventscript in `ovl_En_Dnh`?

Reply:
Correct and correct. I double checked on tharo's msgevent branch.

Reply:
Makes sense. I wonder if the name could be `WEEKEVENTREG_TALKED_EMPTY_BOOTH_BOAT_HOUSE` (see description `Koume in boat house`) or `WEEKEVENTREG_TALKED_EMPTY_BOOTH_TOURIST_INFORMATION` (see scene name)

Reply:
Hmm, in the object we went with `KIOSK_KOUME`. How about `WEEKEVENTREG_TALKED_KIOSK_KOUME_EMPTY`?

Reply:
Nice, these all seem better. Last thought is:
`WEEKEVENTREG_TALKED_KOUME_INJURED`
`WEEKEVENTREG_TALKED_KIOSK_KOUME_EMPTY`

I wonder if the ordering could be made consistent. Like
 `KOUME_INJURED` and `KOUME_KIOSK` or 
`INJURED_KOUME` and `KIOSK_KOUME`

Reply:
Sure though I did change the object file so it also matches Koume Kiosk ordering.